developer_manual/client_apis/OCS/ocs-api-overview.rst

```diff
@@ -114,8 +165,12 @@ Values of the theming app are exposed though the capabilities API, allowing clie
         <url>https://nextcloud.com</url>
         <slogan>A safe home for all your data</slogan>
         <color>#0082c9</color>
+	<color-text>#ffffff</color-text>
```

Comment:
identation

Reply:
This was due to the tabs/spaces mix. Should be ok now.